decidim-proposals/app/helpers/decidim/proposals/proposal_votes_helper.rb

```diff
@@ -22,6 +22,26 @@ def vote_button_classes(from_proposals_list)
         return "small" if from_proposals_list
         "expanded button--sc"
       end
+
+      # Check if the vote limit is enabled for the current feature
+      #
+      # Returns true if the vote limit is enabled
+      def vote_limit_enabled?
+        current_user && feature_settings.vote_limit.present? && feature_settings.vote_limit > 0
+      end
+
+      # Return the remaining votes for a user if the current feature has a vote limit
+      #
+      # user - A User object
+      #
+      # Returns a number with the remaining votes for that user
+      def remaining_votes_count_for(user)
+        if vote_limit_enabled?
```

Comment:
This method can return `nil` if `votes_limit_enabled? === false`, which is weird to me when you are calculating a count. Maybe `return 0 unless vote_limit_enabled?`? What do you think?